uint8 light1_status
uint8 light2_status
uint8 EMPTY=0
uint8 RED=1
uint8 GREEN=2
uint8 YELLOW=3
